test/conformance/tests/sriov_operator.go

```diff
@@ -681,292 +424,559 @@ var _ = Describe("[sriov] operator", func() {
+			})
+		})
 
-				Eventually(func() int64 {
-					testedNode, err := clients.Nodes().Get(node, metav1.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					resNum, _ := testedNode.Status.Allocatable["openshift.io/apivolresource"]
-					capacity, _ := resNum.AsInt64()
-					return capacity
-				}, 3*time.Minute, time.Second).Should(Equal(int64(5)))
+		Context("IPv6 configured secondary interfaces on pods", func() {
+			// 25874
+			It("should be able to ping each other", func() {
+				ipv6NetworkName := "test-ipv6network"
+				testNode := sriovInfos.Nodes[0]
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(node)
 
-				sriovNetwork := &sriovv1.SriovNetwork{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "test-apivolnetwork",
-						Namespace: operatorNamespace,
-					},
-					Spec: sriovv1.SriovNetworkSpec{
-						ResourceName:     "apivolresource",
-						IPAM:             `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`,
-						NetworkNamespace: namespaces.Test,
-					}}
-				err = clients.Create(context.Background(), sriovNetwork)
+				ipam := `{"type": "host-local","ranges": [[{"subnet": "3ffe:ffff:0:01ff::/64"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`
+				err = network.CreateSriovNetwork(clients, sriovDevice, ipv6NetworkName, namespaces.Test, operatorNamespace, resourceName, ipam)
 				Expect(err).ToNot(HaveOccurred())
-
 				Eventually(func() error {
 					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
-					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: "test-apivolnetwork", Namespace: namespaces.Test}, netAttDef)
+					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: ipv6NetworkName, Namespace: namespaces.Test}, netAttDef)
 				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 
-				podDefinition := pod.DefineWithNetworks([]string{sriovNetwork.Name})
-				created, err := clients.Pods(namespaces.Test).Create(podDefinition)
+				pod := createTestPod(testNode, []string{ipv6NetworkName})
+				ips, err := network.GetSriovNicIPs(pod, "net1")
 				Expect(err).ToNot(HaveOccurred())
+				Expect(ips).NotTo(BeNil(), "No sriov network interface found.")
+				Expect(len(ips)).Should(Equal(1))
+				for _, ip := range ips {
+					pingPod(ip, testNode, ipv6NetworkName)
+				}
+			})
+		})
+	})
 
-				var runningPod *corev1.Pod
-				Eventually(func() corev1.PodPhase {
-					runningPod, err = clients.Pods(namespaces.Test).Get(created.Name, metav1.GetOptions{})
+	Describe("Custom SriovNetworkNodePolicy", func() {
+
+		BeforeEach(func() {
+			err := namespaces.Clean(operatorNamespace, namespaces.Test, clients)
+			Expect(err).ToNot(HaveOccurred())
+			waitForSRIOVStable()
+		})
+
+		Describe("Configuration", func() {
+
+			Context("SR-IOV network config daemon can be set by nodeselector", func() {
```

Comment:
This does not need a particular node policy, can be moved in the other section

Reply:
let's do this in another pr